src/main/kotlin/racing_car/domain/race/RaceGame.kt

```diff
@@ -0,0 +1,22 @@
+package racing_car.domain.race
+
+import racing_car.domain.car.Car
+
+class RaceGame(private val carList: List<Car>) {
```

Comment:
List가 특정 도메인을 나타내는게 아니라면, 변수명은 복수형으로 사용해주시면 좋을 것 같아요.
```suggestion
class RaceGame(private val cars: List<Car>) {
```